### ISSUE TITLE
Serverless variables not expanded in secretsFile

### ISSUE THREAD

**Issue by acojocaru** [SC5/serverless-kms-secrets#5]
My configuration:
```
custom:
  serverless-kms-secrets:
    secretsFile: kms-secrets.${opt:region, self:provider.region}.${opt:stage, self:provider.stage}.yml
  kmsSecrets: ${file(kms-secrets.${opt:region, self:provider.region}.${opt:stage, self:provider.stage}.yml)}
```
`kms-secrets.us-east-1.dev.yml` exists and it contains a valid `keyArn`.

Action: 
`sls encrypt -n VAR -v test --region us-east-1 --stage dev`

Expected result:
The variable is encrypted and written in `kms-secrets.us-east-1.dev.yml`.

Actual result:
`Serverless: No config file kms-secrets.${opt:region, self:provider.region}.${opt:stage, self:provider.stage}.yml and no keyid specified`

Looks like the Serverless variables are not expanded at all. Is this an issue with my configuration?

Thanks,
Alex

**Comment by artoliukkonen** [comment]
This is broken on serverless >=1.13. 

serverless@1.12 works fine. #8 seems to fix this issue for newer versions, but until it's merged you can try using older version of serverless.

**Comment by mpuittinen** [comment]
@acojocaru @artoliukkonen Fixed in 0.9.6 (Thanks @martinheidegger )
